events/ME_Hungarian_Revolution_Events.txt

```diff
@@ -0,0 +1,531 @@
+				type = all
+				owned_by = ROOT
+			}
+		}
+		NOT = { has_country_flag = me_hab_bohemia_appeased }
+	}
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.3.a
+		
+		if = {
+			limit = {
+				bohemia_area = {
+					type = all
+					owned_by = ROOT
+				}
+			}
+			bohemia_area = {
+				spawn_rebels = {
+					type = nationalist_rebels
+					size = 3
+				}
+			}
+		}
+		if = {
+			limit = {
+				moravia_area = {
+					type = all
+					owned_by = ROOT
+				}
+			}
+			moravia_area = {
+				spawn_rebels = {
+					type = nationalist_rebels
+					size = 3
+				}
+			}
+		}
+	}
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.3.b
+		
+		if = {
+			limit = {
+				bohemia_area = {
+					type = all
+					owned_by = ROOT
+				}
+			}
+			bohemia_area = {
+				add_local_autonomy = 100
+			}
+		}
+		if = {
+			limit = {
+				moravia_area = {
+					type = all
+					owned_by = ROOT
+				}
+			}
+			moravia_area = {
+				add_local_autonomy = 100
+			}
+		}
+		custom_tooltip = me_hab_bohemia_is_appeased
+		set_country_flag = me_hab_bohemia_appeased
+		add_stability = 1
+	}
+}
+
+country_event = {
+	id = ME_Hungarian_Revolution_Events.4
+	title = ME_Hungarian_Revolution_Events.4.t
+	desc = ME_Hungarian_Revolution_Events.4.d
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.4.a
+		
+		set_country_flag = me_hab_austria_hungary
+		override_country_name = AUSTRIA_HUNGARY
+		add_government_reform = austria_hungary_reform
+	}
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.4.b
+		
+		every_owned_province = {
+			limit = {
+				OR = {
+					culture = hungarian
+					culture = slovak
+					culture = transylvanian
+				}
+			}
+			spawn_rebels = {
+				type = nationalist_rebels
+				size = 3
+			}
+		}
+	}
+}
+
+country_event = {
+	id = ME_Hungarian_Revolution_Events.5
+	title = ME_Hungarian_Revolution_Events.5.t
+	desc = ME_Hungarian_Revolution_Events.5.d
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.5.a
+		
+		set_country_flag = me_hab_austria_hungary
+		override_country_name = AUSTRIA_HUNGARY
+		#gov reform
+	}
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.5.b
+		
+		random_owned_province = {
+			limit = {
+				NOT = { culture_group = ROOT }
+			}
+			spawn_rebels = {
+				type = nationalist_rebels
+				size = 3
+			}
+		}
+		random_owned_province = {
+			limit = {
+				NOT = { culture_group = ROOT }
+			}
+			spawn_rebels = {
+				type = nationalist_rebels
+				size = 3
+			}
+		}
+		random_owned_province = {
+			limit = {
+				NOT = { culture_group = ROOT }
+			}
+			spawn_rebels = {
+				type = nationalist_rebels
+				size = 3
+			}
+		}
+	}
+}
+
+country_event = {
+	id = ME_Hungarian_Revolution_Events.6
+	title = ME_Hungarian_Revolution_Events.6.t
+	desc = ME_Hungarian_Revolution_Events.6.d
+	
+	is_triggered_only = yes
+	
+	option = {
+		name = ME_Hungarian_Revolution_Events.6.a
+		
+		add_stability = 1
+		add_legitimacy_equipvalent = { amount -40 }
```

Comment:
1) lacks =
2) I think in order to work, you need to have republican_tradition as well

Reply:
I tested, it doesn't seem like it does